[2025-09-24T18:44:10.421+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres_dag.create_users_table manual__2025-09-24T18:44:07.960057+00:00 [queued]>
[2025-09-24T18:44:10.428+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres_dag.create_users_table manual__2025-09-24T18:44:07.960057+00:00 [queued]>
[2025-09-24T18:44:10.428+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T18:44:10.436+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_users_table> on 2025-09-24 18:44:07.960057+00:00
[2025-09-24T18:44:10.442+0000] {standard_task_runner.py:60} INFO - Started process 94 to run task
[2025-09-24T18:44:10.445+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_postgres_dag', 'create_users_table', 'manual__2025-09-24T18:44:07.960057+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpy7m53_sa']
[2025-09-24T18:44:10.451+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask create_users_table
[2025-09-24T18:44:10.522+0000] {task_command.py:423} INFO - Running <TaskInstance: test_postgres_dag.create_users_table manual__2025-09-24T18:44:07.960057+00:00 [running]> on host d59ad7cc807c
[2025-09-24T18:44:10.597+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_postgres_dag' AIRFLOW_CTX_TASK_ID='create_users_table' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T18:44:07.960057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-24T18:44:07.960057+00:00'
[2025-09-24T18:44:10.598+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS public.users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        
[2025-09-24T18:44:10.604+0000] {base.py:83} INFO - Using connection ID 'postgres' for task execution.
[2025-09-24T18:44:10.617+0000] {base.py:83} INFO - Using connection ID 'postgres' for task execution.
[2025-09-24T18:44:10.620+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS public.users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2025-09-24T18:44:10.626+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_postgres_dag, task_id=create_users_table, execution_date=20250924T184407, start_date=20250924T184410, end_date=20250924T184410
[2025-09-24T18:44:10.657+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T18:44:10.675+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
